certified-connectors/Tesseron ASM Ticket/apiDefinition.swagger.json

```diff
@@ -0,0 +1,819 @@
+                  "enum": [
+                    true,
+                    false
+                  ],
+                  "default": false
+                },
+                "internalOption": {
+                  "type": "boolean",
+                  "description": "Specify if this is an internal ticket",
+                  "title": "Internal Option",
+                  "enum": [
+                    true,
+                    false
+                  ],
+                  "default": false
+                },
+                "urgencyType": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Select the urgency of the ticket",
+                  "title": "Urgency",
+                  "enum": [
+                    0,
+                    1,
+                    2
+                  ],
+                  "default": 1
+                },
+                "effectsType": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Select the effects of the ticket",
+                  "title": "Effects",
+                  "default": 1,
+                  "enum": [
+                    0,
+                    1,
+                    2
+                  ]
+                },
+                "serviceContractId": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Enter the ID of the service contract",
+                  "title": "Service Contract ID"
+                },
+                "delegatedTicketEditor": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Enter the editor of the ticket",
+                  "title": "Ticket Editor"
+                },
+                "enterpriseId": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Specify the ID of the business partner",
+                  "title": "Enterprise ID"
+                },
+                "contactId": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Select the main contact of the ticket",
+                  "title": "Contact ID"
+                }
+              },
+              "required": [
+                "areaId",
+                "effectsType",
+                "enterpriseId",
+                "entranceType",
+                "internalOption",
+                "privateOption",
+                "releasedOption",
+                "ticketHeader",
+                "ticketText",
+                "ticketType",
+                "urgencyType"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/CreateTicketPosition": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default"
+          },
+          "200": {
+            "description": "sucess",
```

Comment:
@developmentLL please make sure that descriptions and titles are grammatically correct and are free of typographical errors. You can refer to [English strings guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).